dashboard/src/utilities/tablePagination.ts

```diff
@@ -0,0 +1,34 @@
+export const generatePageNumbers = (pageCount: number, currentPage: number) => {
+  const pageNumbers = [];
+
+  // If there are less than 7 pages, show all pages buttons
+  // otherwise show only 5 pages buttons
+  if (pageCount <= 7) {
+    for (let i = 1; i <= pageCount; i++) {
+      pageNumbers.push(i);
+    }
```

Comment:
For creating arrays, you could use a bit neater methods from: https://stackoverflow.com/a/33352604

```js
// Three methods:
Array.from(Array(10).keys()) 
[...Array(10).keys()]
Array.from({length: 10}, (_, i) => i + 1)
```